.storybook/stories/stories_api_pattern_guidelines.mdx

```diff
@@ -0,0 +1,7 @@
+import {Meta} from '@storybook/addon-docs/blocks';
+import Markdown from 'markdown-to-jsx';
+import ApiPatternGuidelines from '../../API_PATTERN_GUIDELINES.md';
+
+<Meta title="Welcome|API Pattern Guidelines" />
+
+<Markdown>{ApiPatternGuidelines}</Markdown>
```

Comment:
Yeah it would be...I looked into it for a bit did not see any options for it: https://github.com/probablyup/markdown-to-jsx#parsing-options. They do have a few `GFM` references in their docs so _not sure_ if this is using `GFM` internally? I can look into it more. 

@anicholls An alternative could be to convert these markdown files to `MDX`. Importing them would be more stream lined then. LMK if you would be ok with this. I haven't actually tested how github renders `MDX` so we could potentially run into rendering issues when you view these files on `github.com`

Reply:
@chad Here's a preview: https://github.com/Workday/canvas-kit/blob/c71f0b5ddb648f237718f8e3243d23f2313dcc1b/modules/card/react/stories/stories.mdx